### PR TITLE
Add docs for Xiaomi Miscale v1 and v2

### PR DESCRIPTION
**Pull request by dckiller51**
## Description:
Doc for Scale Xiaomi Miscale v1 ans v2

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#1368<esphome PR number goes here>

## Checklist:

  - [x] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [x] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/sensor/xiaomi_miscale.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
+Xiaomi Miscale Sensors
+========================
+
+.. seo::
+    :description: Instructions for setting up Xiaomi Miscale bluetooth-based sensors in ESPHome.
+    :image: xiaomi_miscale.jpg
+    :keywords: Xiaomi, BLE, Bluetooth, XMTZC01HM, XMTZC04HM
+
+The ``xiaomi_miscale`` sensor platform lets you track the output of Xiaomi Bluetooth Low Energy devices using the :doc:`/components/esp32_ble_tracker`. This component will track, for example, the weight of the device every time the sensor sends out a BLE broadcast. Contrary to other implementations, ``xiaomi_miscale`` listense passively to advertisement packets and does not pair with the device. Hence ESPHome has no impact on battery life.
```
Comment (jesserockz) [inline components/sensor/xiaomi_miscale.rst]:
```suggestion
The ``xiaomi_miscale`` sensor platform lets you track the output of Xiaomi Bluetooth Low Energy devices using the :doc:`/components/esp32_ble_tracker`. This component will track, for example, the weight of the device every time the sensor sends out a BLE broadcast. Contrary to other implementations, ``xiaomi_miscale`` listens passively to advertisement packets and does not pair with the device. Hence ESPHome has no impact on battery life.
```

File: components/sensor/xiaomi_miscale2.rst [hunk 2]
```diff
@@ -0,0 +1,149 @@
+Xiaomi Miscale2 Sensors
+========================
+
+.. seo::
+    :description: Instructions for setting up Xiaomi Miscale2 bluetooth-based sensors in ESPHome.
+    :image: xiaomi_miscale2.jpg
+    :keywords: Xiaomi, BLE, Bluetooth, XMTZC02HM, XMTZC05HM
+
+The ``xiaomi_miscale2`` sensor platform lets you track the output of Xiaomi Bluetooth Low Energy devices using the :doc:`/components/esp32_ble_tracker`. This component will track, for example, the weight and the impedance of the device every time the sensor sends out a BLE broadcast. Contrary to other implementations, ``xiaomi_miscale2`` listense passively to advertisement packets and does not pair with the device. Hence ESPHome has no impact on battery life.
```
Comment (jesserockz) [inline components/sensor/xiaomi_miscale2.rst]:
```suggestion
The ``xiaomi_miscale2`` sensor platform lets you track the output of Xiaomi Bluetooth Low Energy devices using the :doc:`/components/esp32_ble_tracker`. This component will track, for example, the weight and the impedance of the device every time the sensor sends out a BLE broadcast. Contrary to other implementations, ``xiaomi_miscale2`` listens passively to advertisement packets and does not pair with the device. Hence ESPHome has no impact on battery life.
```

File: index.rst [hunk 3]
```diff
@@ -126,7 +126,6 @@ Sensor Components
     INA219, components/sensor/ina219, ina219.jpg
     INA226, components/sensor/ina226, ina226.jpg
     INA3221, components/sensor/ina3221, ina3221.jpg
-    Inkbird IBS-TH1 Mini, components/sensor/inkbird_ibsth1_mini, inkbird_isbth1_mini.jpg
```
Comment (jesserockz) [inline index.rst]: Please check your commits to not remove unrelated lines

File: index.rst [hunk 4]
```diff
@@ -217,7 +218,6 @@ Output Components
     TLC59208F, components/output/tlc59208f, tlc59208f.jpg
     MY9231/MY9291, components/output/my9231, my9231.svg
     SM16716, components/output/sm16716, sm16716.svg
-    MCP4725, components/output/mcp4725, mcp4725.jpg
```
Comment (jesserockz) [inline index.rst]: Please check your commits to not remove unrelated lines

**Review by jesserockz — CHANGES_REQUESTED** [review]
A few changes to make please

File: components/sensor/xiaomi_miscale.rst (hunk 1 above)
Comment (dckiller51) [reply components/sensor/xiaomi_miscale.rst]: update ok

File: components/sensor/xiaomi_miscale2.rst (hunk 2 above)
Comment (dckiller51) [reply components/sensor/xiaomi_miscale2.rst]: update ok

File: index.rst (hunk 4 above)
Comment (dckiller51) [reply index.rst]: update ok

File: index.rst (hunk 3 above)
Comment (dckiller51) [reply index.rst]: update ok

**Review by jesserockz — APPROVED** [review]
Good to go. Thanks
